Makefile

```diff
@@ -2,7 +2,7 @@
 DEIS_REGISTRY ?= $(DEV_REGISTRY)
 IMAGE_PREFIX ?= deis
 COMPONENT ?= workflow
-VERSION ?= git-$(shell git rev-parse --short HEAD)
+VERSION ?= $(shell git rev-parse --short HEAD)
```

Comment:
would you mind keeping this as `git-$(shell git rev-parse --short HEAD)` and change the client Makefile to match so it matches https://github.com/deis/prototype-repo/blob/master/Makefile#L12-L14? :)